apps/src/code-studio/utils.js

```diff
@@ -48,3 +48,17 @@ export function updateQueryParam(param, value) {
   }
   window.history.pushState(null, document.title, newLocation);
 }
+
+/**
+ * We indicate that a page is Rtl by setting the dir attribute on the root element
```

Comment:
Where do we set the dir attribute on the root element? I'm not a huge fan of mirroring data between the DOM and the redux store if we don't have to. There should be one source of truth and everything should refer to that. So either we get rid of the dir attribute on the root element and replace the code that adds it with a call to `getStore().dispatch(setRtl()))`, or we leave it there and just use the below `isRtl()` function everywhere instead of `state.isRtl`.

The benefit to putting it in redux IMO is that it allows us to trivially add a feature down the road where you can switch back and forth between rtl and ltr languages without reloading the page. Whether we will ever care to add such a feature however is debatable.